src/chain_sync/network_context.rs

```diff
@@ -173,9 +171,7 @@ where
                         || header_len != msg.secp_msg_includes.len()
                     {
                         tracing::warn!(
-                            "header_len: {header_len}, msg.bls_msg_includes.len(): {}, msg.secp_msg_includes.len(): {}",
-                            msg.bls_msg_includes.len(),
-                            msg.secp_msg_includes.len()
+                            %header_len, msg.bls_msg_includes.len = %msg.bls_msg_includes.len(), msg.secp_msg_includes.len = %msg.secp_msg_includes.len()
```

Comment:
nit: kinda nice, but confusing. I would never be able to deduce what `msg.bls_msg_includes.len = %msg.bls_msg_includes.len()` does without the diff.

Reply:
I think this just comes down to knowing `tracing::event`'s API, like knowing `format!` or `printf` 🤷

Reply:
Perhaps it would be easier to understand if you did something like `bls_msg_includes_len = ...`. It's not blocking, but this is not reading well nor should reading the code require one to know `tracing` API specifics. `format!` and `printf` are intuitive, while this just looks like you are assigning a value to a property of an object.

Reply:
While I agree that the `tracing` API is not particularly obvious, it seems a standard in the Rust ecosystem https://blessed.rs/crates#section-common-subsection-logging. As such, we should learn how to utilize it and get familiar with it. I recommend tinkering a bit to get a hold of it, what results are produced etc. We can't stay in the `printf` world forever. :t-rex: